petprep/workflows/pet/confounds.py

```diff
@@ -47,8 +47,6 @@
 
 def init_pet_confs_wf(
     mem_gb: float,
-    metadata: dict,
-    regressors_all_comps: bool,
     regressors_dvars_th: float,
     regressors_fd_th: float,
     freesurfer: bool = False,
```

Comment:
This is only used here:
https://github.com/jhlegarreta/petprep/blob/12cdb163f29b2057b092512fb6508cfe4c19be27/petprep/workflows/pet/confounds.py#L146C5-L150

but the `gm_desc` variable is not used. Should the parameter be removed?

Reply:
Yeah - I believe so. Thanks @jhlegarreta .

Reply:
The same code exists in `fMRIPrep` with no apparent use of `gm_desc`. Pinging @effigies to hopefully cast some more light and tell if changes are needed there as well. Thanks.